### PR TITLE
Pages/cart

### PR DESCRIPTION
**Pull request by manish591**
#### Added Cart Management page for e-commerce. 
Here is the code sandbox link:  https://codesandbox.io/s/github/manish591/Novee/tree/pages/Cart

### CODE REVIEW

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+  margin-block-start: 5rem;
+}
+
+.cart__wrapper {
+  inline-size: min(98%, 1000px);
+  margin-inline: auto;
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 1.5rem;
+  grid-row-gap: 1rem;
```
Comment (surajmore98) [inline css/cart.css]: you can add grid-row-gap value with gap property "gap: row-gap column-gap;"

File: css/cart.css [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+}
+
+.summary-item__picture {
+  inline-size: 50px;
+  aspect-ratio: 1;
+  background-color: var(--surface4);
+}
+
+.price-detail {
+  margin-block-start: 0.5rem;
```
Comment (surajmore98) [inline css/cart.css]: you can use shorthand property margin-block

File: css/cart.css [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+  margin-block-end: 2rem;
+  border-block-end: 1px solid var(--surface4);
+}
+
+.summary__price-details {
+  padding-block-start: 0.5rem;
```
Comment (surajmore98) [inline css/cart.css]: can use padding-block

File: css/cart.css [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+    border: 1px solid var(--clr-primary);
+    font-size: unset;
+  }
+
+  .cart-action__btns {
+    display: block;
+  }
+
+  .cart-action__icon {
+    display: none !important;
```
Comment (romabulani) [inline css/cart.css:320]: Just a suggestion, avoid using important

**Review by romabulani — APPROVED** [review]
Rest all looks good to merge

**Review by surajmore98 — APPROVED** [review]

**Pull request merged**